dev/ci/util_free_space.sh

```diff
@@ -23,12 +23,6 @@ set -eux
 
 if [ "${GITHUB_ACTIONS}" = "true" ]; then
   df -h
-  echo "::group::/usr/local/*"
-  du -hsc /usr/local/*
-  echo "::endgroup::"
-  echo "::group::/usr/local/bin/*"
-  du -hsc /usr/local/bin/*
-  echo "::endgroup::"
```

Comment:
Are these steps very time-consuming?

Reply:
I saw the
I once saw that the free_space step took several minutes to complete, which should have been caused by IO generated by the `du` command. After removing these unnecessary `du` commands, the free_space step only takes a few seconds.

Reply:
Got it.